devtools/create_project/xcode.cpp

```diff
@@ -25,6 +25,7 @@
 
 #include <fstream>
 #include <algorithm>
+#include <sys/stat.h>
```

Comment:
> 1. Unconditionally specify all the directories in search paths.

Does this even work? It looks like it's for static library linking, which takes a full path. I doubt we can simply supply every library and the non-existent ones only cause warnings.

> 2. Add --prefix flag to the generator that will override the path search if it is specified.

Might work, still doesn't help when we pre-generate project files AFAICT.

> I guess I also don’t know how to get cross-platform code for finding a directory into create_project. I tried to include common/fs.cpp but it ends up requiring dependencies of irrelevant things like GUI::EventRecorder because it uses flags for the entire build. I don’t know about how to make make do what I want it to do, and I don’t even know if this approach would be acceptable anyway

Yeah, you will need to implement it yourself in devtools/ code. ScummVM main code cannot be linked to devtools binaries.